_data/current.yml

```diff
@@ -424,6 +408,11 @@
   dates: "August 30-September 2, 2021"
   url: https://www.autotestcon.com?utm_source=testingconferences
 
+- name: QA Fest 2021
+  location: Kyiv, Ukraine
+  dates: "September, 2019"
```

Comment:
says 2019 instead of 2021. But that's an easy fix.

Reply:
fixed